api/Api/Aether/Lobby/LobbyClient.cs

```diff
@@ -367,27 +366,26 @@ internal class LobbyClientHandler : IDisposable {
                     var startRead = DateTime.UtcNow;
                     while (bytesRead != msgSize && (DateTime.UtcNow - startRead).TotalMilliseconds < READ_TIMEOUT_MS) {
                         bytesRead += stream.Read(msgBuf, bytesRead, msgSize - bytesRead);
-                        Thread.Sleep(10);
```

Comment:
Is reading from a network stream particularly expensive? I don't see why we needed a `Thread.Sleep` and removing it completely removed the lag.

Reply:
Well if removing it improves the lag then its the sleep that seems to be more expensive.